pages/persons/view/person_current_view_page.py

```diff
@@ -0,0 +1,229 @@
+                "[ng-model='entirePerson.addresses.postAddresses.adminUnitId']>div:nth-child(1) span:nth-child(2) span")
+    POST_ADRESS_LEVEL_2 = (By.CSS_SELECTOR, ".row.form-group.ng-scope>div>label+div"
+                "[ng-model='entirePerson.addresses.postAddresses.adminUnitId']>div:nth-child(2) span:nth-child(2) span")
+    POST_ADRESS_LEVEL_3 = (By.CSS_SELECTOR, ".row.form-group.ng-scope>div>label+div"
+                "[ng-model='entirePerson.addresses.postAddresses.adminUnitId']>div:nth-child(3) span:nth-child(2) span")
+    POST_ADRESS_LEVEL_4 = (By.CSS_SELECTOR, ".row.form-group.ng-scope>div>label+div"
+                "[ng-model='entirePerson.addresses.postAddresses.adminUnitId']>div:nth-child(4) span:nth-child(2) span")
+    POST_ADRESS_LEVEL_5 = (By.CSS_SELECTOR, ".row.form-group.ng-scope>div>label+div"
+                "[ng-model='entirePerson.addresses.postAddresses.adminUnitId']>div:nth-child(5) span:nth-child(2) span")
+    POST_ADRESS_LEVEL_6 = (By.CSS_SELECTOR, ".row.form-group.ng-scope>div>label+div"
+                "[ng-model='entirePerson.addresses.postAddresses.adminUnitId']>div:nth-child(6) span:nth-child(2) span")
+    POST_ADRESS_LEVEL_7 = (By.CSS_SELECTOR, ".row.form-group.ng-scope>div>label+div"
+                "[ng-model='entirePerson.addresses.postAddresses.adminUnitId']>div:nth-child(7) span:nth-child(2) span")
+
+    # Exact post addresses
+    arr_titles_post_exact = ["POST_ZIPCODE", "POST_STREET_TYPE", "POST_STREET",
+                            "POST_HOME", "POST_APARTMENT"]
+    POST_ZIPCODE = (By.ID, "inputZipCodePost")
+    POST_STREET_TYPE = (By.ID, "inputStreetTypePost")
+    POST_STREET = (By.ID, "inputStreetPost")
+    POST_HOME = (By.ID, "inputHousePost")
+    POST_APARTMENT = (By.ID, "inputApartmentPost")
+
+    # >>>>>>>>>>>>>>>>>>>>>>>>>>> Global information about person <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
+
+    @property
+    def get_fio_ukranian(self):
+        return self.driver.find_element(*self.FIO_PERSON_UKRANIAN)
+
+    @property
+    def get_fio_english(self):
+        return self.driver.find_element(*self.FIO_PERSON_ENGLISH)
+
+    def get_serial_person_record(self):
+        text_with_colon = self.driver.find_element(*self.SERIAL_PERSONAL_RECORD).text.encode("cp1251")
+        value_without_spaces = self.__get_text_splited_by_colon(text_with_colon)
+        return value_without_spaces
+
+    def get_number_person_record(self):
+        text_with_colon = self.driver.find_element(*self.NUMBER_PERSONAL_RECORD).text.encode("cp1251")
+        value_without_spaces = self.__get_text_splited_by_colon(text_with_colon)
+        return value_without_spaces
+
+    # >>>>>>>>>>>>>>>>>>>>>>>>>>> Main information about person <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
+
+    def get_dict_main_info_about_person(self):
+        arr_values = [self.TYPE_OF_PERSON, self.DATE_OF_BIRTH, self.RESIDENT, self.SEX, self.MARITAL_STATUS,
```

Comment:
you need to have only one data structure with the names of the fields. Instead you have 3: arr_values, arr_titles_main and titles.
You need to refactor this! :)